test/test_functionalization.py

```diff
@@ -1542,6 +1543,87 @@ def forward(self, x_1):
     add = torch.ops.aten.add.Tensor(view_1, 1);  view_1 = None
     return add""")
 
+    def test_python_functionalization_zero_tensor(self):
+        def f(x):
+            y = torch.ops.aten._efficientzerotensor([4])
+            out = x + y
+            out.mul_(2)
+            return out
+        x = torch.randn(4)
+        out_ref = f(x)
+        out_test = dispatch_functionalize(f)(x)
+        out_test_cpp = _functionalize(f, reapply_views=True, crossref=False, skip_input_mutations=True)(x)
+        self.assertEqual(out_ref, out_test)
+        self.assertEqual(out_ref, out_test_cpp)
+        if not IS_WINDOWS:
```

Comment:
What does windows have to do with it?

Reply:
whoops I'll take it out - the test right above it had the same check, although looking at the PR history I realize now that this is only for `instance_norm` (strangely when we trace instance_norm, the nodes in the FX graph show up in a slightly different order on windows, breaking expect tests) https://github.com/pytorch/pytorch/pull/88697/files#diff-e302079b2ae45f6ada1a3cf98a49dfe5b518efbb51d2299de9d494246d4f74cbR1238